Makefile

```diff
@@ -222,6 +222,9 @@ docker-tag-with-git-tag:
 ## Testing
 ## -------
 
+# FIXME(#4364) test_thread_load_statistic is a flaky test - on CI gets 0 cpu usages, but passes locally.
+SKIPPED_TESTS := --skip test_thread_load_statistic
```

Comment:
So, I think this check won't help - this will only ignore it locally where it currently passes and not on CI where it is flaky